.pre-commit-config.yaml

```diff
@@ -3,4 +3,3 @@ repos:
   rev: stable
   hooks:
   - id: black
-    language_version: python3.6
```

Comment:
Was there a reason this was removed?

Reply:
The precommit was failing for me when using Python 3.7. I think that removing this line, the Python version was not forced. Correct me if I'm wrong though.

Reply:
Ahh, got it. We'll want to keep this as it was before since that ensures we're all running it using the same python version. If this is causing you issues on the commit side of things, you could temporarily remove this and run `black` manually to make sure that files are formatted correctly. Then you can add it back in so that it works for everyone else. Thoughts?

Reply:
I'm afraid that I cannot commit if I remove this line